pkg/flash/flash_linux.go

```diff
@@ -104,7 +106,7 @@ func (f *Flash) Size() int64 {
 
 const maxTransferSize = 4096
 
-func min(x, y int64) int64 {
```

Comment:
For some reason, `min` is now a language built-in since go 1.21. https://go.dev/play/p/rZtnXH0hPtN

Reply:
Really, I had to do a double-take. They added min/max functions to the language itself (https://go.dev/ref/spec#Min_and_max) and not the just standard library.

Reply:
finally.

Reply:
I bet they added it because everybody write them

Reply:
Anyway, we have to leave this here until the CI runs on 1.21 uniformly. We're stuck on 1.20 on a few targets still.